sitemap.html

```diff
@@ -1,40 +1,52 @@
-	<head>
-			<title>Sitemap</title>
-			<meta name="description" content="Sitemap" />
-			<meta name="keywords" content="Sitemap" />
-			<meta name="generator" content="Mystemapgenerator.com HTML Sitemap" />
-			<meta charset="utf-8" />
-			<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=yes" />
-			<link rel="stylesheet" href="css/sitemapStyles.css">
-			
-			<link rel="apple-touch-icon" sizes="180x180" href="favicon_package\apple-touch-icon.png">
-			<link rel="icon" type="image/png" sizes="32x32" href="favicon_package\favicon-32x32.png">
-			<link rel="icon" type="image/png" sizes="16x16" href="favicon_package\favicon-16x16.png">
-			<link rel="manifest" href="favicon_package\site.webmanifest">
-			<link rel="mask-icon" href="favicon_package\safari-pinned-tab.svg" color="#3caea3">
-			<meta name="msapplication-TileColor" content="#2b5797">
-			<meta name="theme-color" content="#ffffff">
-
-	</head>
-	<body>
-		<h1>Sitemap</h1>
-		<div class="section">
-			<p><a href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/">Home page</a></p>
-			<div><a href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/#about">About Page</a></div>
-
-			<div><a href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/team.html">Teams Page</a></div>
-			<div><a href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/#footer">Contact Page</a></div>
-			<div><a href="https://www.codechef.com/ide">Code Editor</a></div>
-			<div><a href="https://www.geeksforgeeks.org/data-structure-gq/">Quizes</a></div>
-			<div><a href="https://github.com/Algo-Phantoms/Algo-Tree">DSA Guidance</a></div>
-			<div><a href="https://github.com/Algo-Phantoms">GitHub</a></div>	
-
-			<!-- For new link just add anchor tag within the div tag  -->
-		</div>	
-		<div class="footerinfo">Created by 
-			<a href="https://www.cratonik.com/" target="_blank">Chaitanya Shimpi</a>
-		</div>
-	</body>
-</html>
+<!DOCTYPE HTML>
+<html lang="en">
+	<head>
+			<title>Sitemap | Algo-Phantoms</title>
+			<meta name="description" content="Sitemap" />
+			<meta name="keywords" content="Sitemap" />
+			<meta name="generator" content="Mystemapgenerator.com HTML Sitemap" />
+			<meta charset="utf-8" />
+			<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=yes" />
+			<link rel="stylesheet" href="css/sitemapStyles.css">
+			<link rel="shortcut icon" href="images/icon.png" type="images/icon.png" />
+	</head>
+	<body>
+		<!--======================= Header starts here =========================-->
+		<div class="heading">
+			<a class="navbar-brand" href="index.html"><img src="images/icon.png" alt="Algo Phantoms" width="60px"
+				height="60px"><span class="brand1"></a>
+			<h1>Algo-Phantoms</h1>
+			<h1 class="note">Sitemap</h1>
+		</div>
+		<!--========================= Site Map Tree ============================-->
+		<div class="section">
+			<p><a href="https://algo-phantoms.github.io/Algo-Phantoms-Frontend/">Home page</a></p>
```

Comment:
sir I changed the link format for the home-page but how I can achieve dynamic links for a section like how can i do it for about us section ?? 
I tried by calling ID but since it was a part for other page , it didnt worked .
please help me .

Reply:
We can give like `index.html#about` this it will work. Check the `team.html` navbar the links over there are present like that only.